core/cmd/shell.go

```diff
@@ -159,11 +159,17 @@ func (n ChainlinkAppFactory) NewApplication(ctx context.Context, cfg chainlink.G
 
 	loopRegistry := plugins.NewLoopRegistry(appLggr, cfg.Tracing())
 
+	// TOML config + env var protected usage of grpc mercury pool
+	var mercuryTlsCertFile *string
+	if os.Getenv("CHAINLINK_MERCURY_USE_GRPC") == "true" {
```

Comment:
Probably better as a config variable

Reply:
I think this is true now that many of our smoke tests are moving to using TOML.